### DISCUSSION TITLE
%E9%A1%B9%E7%9B%AE%E6%80%BB%E7%BB%93/%E8%A1%A8%E9%9D%A2%E7%BC%BA%E9%99%B7%E6%A3%80%E6%B5%8B%E9%A1%B9%E7%9B%AE%E7%9A%84%E6%96%B9%E6%B3%95%E8%AE%BA/

### DISCUSSION THREAD

**Discussion by giscus** [thb1314/thb1314.github.io#6] [Announcements]
# %E9%A1%B9%E7%9B%AE%E6%80%BB%E7%BB%93/%E8%A1%A8%E9%9D%A2%E7%BC%BA%E9%99%B7%E6%A3%80%E6%B5%8B%E9%A1%B9%E7%9B%AE%E7%9A%84%E6%96%B9%E6%B3%95%E8%AE%BA/

https://www.hbblog.cn/%E9%A1%B9%E7%9B%AE%E6%80%BB%E7%BB%93/%E8%A1%A8%E9%9D%A2%E7%BC%BA%E9%99%B7%E6%A3%80%E6%B5%8B%E9%A1%B9%E7%9B%AE%E7%9A%84%E6%96%B9%E6%B3%95%E8%AE%BA/

**Comment by DCC-lzhy** [comment]
现在是去做哪个方向的工作啦？博主

**Reply by thb1314** [reply]
大模型训练相关的

**Reply by DCC-lzhy** [reply]
老哥，稳。这个瑕疵检测赛道太卷了，害，这两年做视觉相关的公司挺难搞的，没订单就完犊子。
